FIDO2_messages.md

```diff
@@ -0,0 +1,21 @@
+## [](#header-1) Communications Between Aux and Main MCU
+This describes the FIDO2 messages sent between Aux MCU and Main MCU.
+
+## [](#header-2) Message Structure and Serial Link Specs 
+Each FIDO2 message is wrapped in the header defined as follows"
+
+| byte 0 - 1         | byte 2 - 3        | byte 4 - 551  |
+|:-------------------|:------------------|:--------------|
+| FIDO2 Message Type | Reserved          | Payload       |
```

Comment:
byte numbering is confusing I think
byte 0-1 should be 0x0009 (FIDO)
byte 2-3 should be "depending on payload"
byte 4-5 should be FIDO2 Message Type
etc...
out of curiosity, any reason why the 2 bytes are reserved?

Reply:
byte 0-1: 0x0009 is in the outer message (aux_mcu_message_t). I am describing the inner (FIDO2 message) which is embedded inside the aux_mcu_message_t.
byte 2-3: For 4 byte alignment. Maybe not needed but always nice to have something reserved if we have room.

Reply:
I'd say either change packet byte numbers or make it explicit that it is the internal number....  
I'd vote for the former to not create any kind of confusion

Reply:
so absolute byte # instead of relative? so instead of starting with byte 0-1, start with whatever absolute byte this inner message start at?

Reply:
i'd vote for absolute indeed :)